nilearn/interfaces/fmriprep/load_confounds_strategy.py

```diff
@@ -152,7 +157,7 @@ def load_confounds_strategy(img_files, denoise_strategy="simple", **kwargs):
         N/A                 N/A            N/A       N/A       True*
         scrubbing True      full*  full   None*         5*    0.2*         \
         3*                  N/A            N/A       N/A       True*
-        compcor   True      full*  N/A    N/A           N/A   N/A          \
+        compcor   True      full*  N/A    None*           N/A   N/A          \
```

Comment:
doc build fails because of this I think

https://github.com/nilearn/nilearn/actions/runs/7558980709/job/20582036244?pr=4225#step:34:991

```suggestion
        compcor   True      full*  N/A    None*         N/A   N/A          \
```